requirements-ci.txt

```diff
@@ -1,6 +1,6 @@
 -r requirements-cython.txt
 flake8==4.0.1
-black==21.10b0
+black>=21.10b0
```

Comment:
According to your reference is should be `>=22.3.0`

Reply:
Thanks for the hint. I fixed it.